cypress/e2e/stack.cy.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress" />
+
+import { login, logout } from './utils'
+
+describe('stack tests', () => {
+    beforeEach(login)
+    afterEach(logout)
```

Comment:
Considering the fact that we probably want to do this for 99% of our tests, we maybe should look into adding a top level `beforeEach` and `afterEach` that does this.

One complication reg. this would be the fact that we also want to add tests for actually logging in and out, and for these tests we _don't_ want to automatically log in and out. I don't know if there's an elegant solution for this, and if not, I guess just having these two lines above at the top of every test is OK. You could even argue it makes the code nice and explicit.